presto-tests/src/test/java/io/prestosql/connector/informationschema/BenchmarkInformationSchema.java

```diff
@@ -138,6 +147,16 @@ public void tearDown()
         }
     }
 
+    private static void emulateMetastoreCall()
+    {
+        try {
+            Thread.sleep(1);
```

Comment:
I no longer think this is going to be useful, notice that count the times how many times metadata (metastore) is called in `io.prestosql.tests.TestInformationSchemaConnector#testLargeData`. Also it would be nice to count each method individually. Testing this way there is no need for benchmark.

What do you think?

Reply:
I would leave it in testng test. Benchmark could be useful for testing the engine in general.